tests/callbacks/test_EnqueuedEventTimer.cpp

```diff
@@ -0,0 +1,78 @@
+#include "gtest/gtest.h"
+
+#include "Kokkos_Core.hpp"
+
+#include "kokkos-utils/callbacks/EnqueuedEventTimer.hpp"
+#include "kokkos-utils/timer/Duration.hpp"
+
+/**
+ * @addtogroup unittests
+ *
+ * Enqueued event timer
+ * --------------------
+ *
+ * This group of tests check the behavior of @c Kokkos::utils::callbacks::EnqueuedEventTimer
+ * and related utilities.
+ */
+
+using execution_space = Kokkos::DefaultExecutionSpace;
+
+namespace Kokkos::utils::tests::callbacks
+{
+
+using namespace Kokkos::utils::callbacks;
+
+struct EnqueuedEventTimerTest : public ::testing::Test
+{
+public:
+    using timer_t = EnqueuedEventTimer<execution_space>;
+
+public:
+    void SetUp() override {
+        this->exec = Kokkos::Experimental::partition_space(execution_space{}, 1)[0];
+        this->timer = timer_t{this->exec};
+    }
+
+protected:
+    execution_space exec {};
+    timer_t timer {};
+};
+
+//! @test Check @ref Kokkos::utils::callbacks::EnqueuedEventTimer.
+TEST_F(EnqueuedEventTimerTest, duration)
+{
+    const auto exec_dev_id = Kokkos::Tools::Experimental::device_id(this->exec);
```

Comment:
Both tests need the `exec_dev_id`. Let's move it to the fixture class ?

Reply:
Done!